### PR TITLE
Modernize and clean up project

### PR DESCRIPTION
**Pull request by adelq**
* Set up testing/CI infrastructure
* Add Python 3 support
* Add `__contains__` to heaps

### CODE REVIEW

File: trees/heap.py [hunk 1]
```diff
@@ -69,6 +68,9 @@ def __iter__(self):
         while len(self) > 0:
             yield self.pop()
 
+    def __contains__(self, item):
+        return item in self._items
```
Comment (Ceasar) [inline trees/heap.py]: I'm pretty sure it should be possible to implement this is `O(sqrt(n))`, which is why I'm guessing I didn't implement this. In any case, better to have this implemented and slow, than not implemented at all.

**Comment by Ceasar** [comment]
This looks great though. Really appreciate it!

File: trees/heap.py (hunk 1 above)
Comment (adelq) [inline trees/heap.py]: Figured that was the case, came to the same conclusion

**Comment by adelq** [comment]
:+1:

File: README.rst [hunk 2]
```diff
@@ -4,8 +4,8 @@ trees
 .. image:: https://badge.fury.io/py/trees.png
     :target: http://badge.fury.io/py/trees
 
-.. image:: https://pypip.in/d/trees/badge.png
-        :target: https://crate.io/packages/trees/
+.. image:: https://travis-ci.org/adelq/trees.svg
+    :target: https://travis-ci.org/adelq/trees
```
Comment (Ceasar) [inline README.rst]: Oops, this is actually an error-- should probably point to my account since I have the original repo.

Comment (Ceasar) [inline README.rst]: Which explains why the Travis CI didn't run.

**Pull request merged**